students/HarveyHerela/lesson07/html_render.py

```diff
@@ -0,0 +1,128 @@
+class Element:
+    """Base class for all HTML tags."""
+    def __init__(self, tag_name='element', content=None, **kwargs):
+        if content is None:
+            self.content = list()
+        else:
+            self.content = [content]
+        tag_style = ''
+        if 'style' in kwargs:
+            tag_style = ' style="{style}"'.format(style=kwargs['style'])
+        self.tag_open = "<{tag}{tag_style}>".format(tag=tag_name, tag_style=tag_style)
+        self.tag_close = "</{tag}>".format(tag=tag_name)
+
+
+    def append(self, content):
+        self.content.append(content)
+
+
+    def render(self, file_out, cur_ind = "", one_line_tag=False):
+        file_out.write(cur_ind + self.tag_open)
```

Comment:
I would include the logic for `OneLineTag` in the `render` method of that subclass, so you don't need the boolean parameter here in the base class